cmd/livepeer/livepeer.go

```diff
@@ -130,47 +153,30 @@ func main() {
 		}
 	}
 
-	if *rinkeby {
-		if !*offchain && !transcoder {
-			if *ethUrl == "" {
-				*ethUrl = "wss://rinkeby.infura.io/ws"
+	/// Setting config options based on specified network
+	if *network != "" {
+		if netw, ok := configOptions[*network]; ok {
+			if *orchestrator {
+				if *ethUrl == "" {
+					*ethUrl = netw.ethUrl
+				}
+				if *ethController == "" {
+					*ethController = netw.ethController
+				}
 			}
-			if *controllerAddr == "" {
-				*controllerAddr = RinkebyControllerAddr
+			if *monitor && *monHost == "" {
+				*monHost = netw.monHost
 			}
-			glog.Infof("***Livepeer is running on the Rinkeby test network: %v***", RinkebyControllerAddr)
-			*datadir = *datadir + "/rinkeby"
+			glog.Infof("***Livepeer is running on the %v*** network: %v***", *network, *ethController)
 		} else {
-			*datadir = *datadir + "/offchain"
+			glog.Infof("***Livepeer is running on the %v*** network", *network)
 		}
-
-		if *monitor && *monhost == "" {
-			*monhost = "http://metrics-rinkeby.livepeer.org/api/events"
-		}
-	} else if *devenv {
-		*datadir = *datadir + "/devenv"
 	} else {
-		if !*offchain && !transcoder {
-			if *ethUrl == "" {
-				*ethUrl = "wss://mainnet.infura.io/ws"
-			}
-			if *controllerAddr == "" {
-				*controllerAddr = MainnetControllerAddr
-			}
-			glog.Infof("***Livepeer is running on the Ethereum main network: %v***", MainnetControllerAddr)
-			*datadir = *datadir + "/mainnet"
-		} else {
-			*datadir = *datadir + "/offchain"
-		}
-		if *monitor && *monhost == "" {
-			*monhost = "http://metrics-mainnet.livepeer.org/api/events"
-		}
+		*network = "offchain"
```

Comment:
Note: need to add ability for users to name their own network, and still be "offchain"

Reply:
True! We can indeed live without that feature. I'll make the default `*network = "offchain"`